go/src/koding/kites/config/config.go

```diff
@@ -104,6 +104,36 @@ func (u *URL) WithPath(paths ...string) *URL {
 	return &URL{URL: &ur}
 }
 
+// StringOrBool implements json.Unmarshaler, which can parse
+// either boolean type or converts string to bool.
+type StringOrBool bool
+
+var _ json.Unmarshaler = (*StringOrBool)(nil)
+
+func (b *StringOrBool) UnmarshalJSON(p []byte) error {
```

Comment:
This type here is explicitly for the purpose of accepting boolean value of its stringified encoding.

`kd config set debug true` - the argument here is always a string, so it must be possible to unmarshal `{"debug": "true"}` onto *config.Konfig value. I could make the field `Debug string` instead, but then I would need to parse it either way in every place I would want to use it.

So the StringOrBool basically implements an union - I was loosely inspired by [IntOrString](https://github.com/kubernetes/kubernetes/blob/81d788dd6e0748e5d53a62c16d933c5f7a0718af/staging/src/k8s.io/client-go/pkg/util/intstr/intstr.go#L43) wrapper.